templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -53,17 +53,14 @@ If you have Homebrew installed and prefer to use it, here's how.
 1. Set up the Zulip Desktop apt repository and its signing key, from a
    terminal:
 
-```
-sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
-echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
-  sudo tee -a /etc/apt/sources.list.d/zulip.list
-```
+        sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
+        echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
+        sudo tee -a /etc/apt/sources.list.d/zulip.list
 
 2. Install the client, from a terminal:
-```
-sudo apt update
-sudo apt install zulip
-```
+
```

Comment:
Is there any purpose for the extra line here, maybe to activate the fenced code block?

Reply:
yes it is

Reply:
Yet there is no extra line at the previous one.

Reply:
all these 3 lines are included in one codeblock
There should be one empty line before and after each code block.